### ISSUE TITLE
Support for cheerio options

### ISSUE THREAD

**Issue by dherges** [zurb/inky#25]
Right now, ``releaseTheKraken()`` replaces ``"`` with ``&quot;`` (and other entties as well).

Issue 1: ``componentFactory`` should use ``elem.html()`` instead of ``$.html(elem)``

Issue 2: it should be possible to pass cherio options to the ``inky()`` function --> already done on master

**Comment by ibarral** [comment]
Hello,

Do you have an example to pass cherio options to the inky() function? I can't make it work.

Thanks!

**Comment by dherges** [comment]
Yes, use it like this with gulp/vinyl:

```js
vfs.src('somefile.html')
  .pipe(inky({cheerio: {decodeEntities: false}}))
  .dest('output')
```

``somefile.html``:
```html
"mytext"
```

output:

```html
&quot;mytext&quot;
```

So, the options are actually passed to ``cheerio.load(input, opts)``, but entities (``"``) go missing in the final output (after ``releaseTheKraken().html()`` ... guess that happens in ``componentFactory.js``

**Comment by dherges** [comment]
I was able to drill down the issue

this input results in decoded entities for the text ``"broken"`` .. output ``&quot;broken&quot;"`` despite passing ``{decodeEntities: false}``. "works" is outputted as ``"works"`` (since its outside the ``<center>`` tag ?).

```html
<table class="body">
  <tr>
    <td class="center" align="center" valign="top">
      <center>
"broken"
      </center>
"works"
    </td>
  </tr>
</table>
```

putting an attribute ``data-parsed`` on the ``<center>`` tag works for both texts. inside and outside the center tag, output is ``"works"``

```html
<table class="body">
  <tr>
    <td class="center" align="center" valign="top">
      <center data-parsed>
"works"
      </center>
"works"
    </td>
  </tr>
</table>
```

**Comment by ibarral** [comment]
Thanks for the update!

But I have a doub't: I have installed Inky using NPM (https://www.npmjs.com/package/inky).

I'm using Gulp to compile the files, and if I pass the cheerio options to Inky, for example:

`.pipe(inky({cheerio: {decodeEntities: false}}))`

It won't work.

This feature works ok from the NPM version?

Thanks!

**Comment by dherges** [comment]
No, the feature was introduced here https://github.com/zurb/inky/commit/9fda83ed637dbd81db95314a988ed61d6b814485#diff-168726dbe96b3ce427e7fedce31bb0bc and is included in the master branch ... 

try ```npm install github:zurb/inky#master``` to get it

**Comment by ibarral** [comment]
Ok, thanks for your help! :)

**Comment by rafibomb** [comment]
@dherges @ibarral Just published the [Foundation for Emails 2.1](http://zurb.com/article/1434/foundation-for-emails-2-1-supercharge-you) release blog post and the release went out this morning! We also started a discussion on the future of Inky and what amazing integrations can be made. I'm sure you have some comments, questions, or ideas: http://foundation.zurb.com/forum/posts/39717-architecting-the-future-of-foundation-for-emails
